README.md

```diff
@@ -1,2 +1,19 @@
 # ssp-operator
 Kubevirt SSP Operator
+
+### Building
+```shell script
+make operator-build
+```
+
+### Uploding to image repository
```

Comment:
Maybe add
```
export IMAGE_REGISTRY=<registry>
export OPERATOR_IMAGE=<image_name>
export IMAGE_TAG=<image_tag>
```
To explicitly show how to modify the image location

Reply:
Maybe add this to the `Building` section actually

Reply:
Done